src/overlays/actors/ovl_Bg_Tobira01/z_bg_tobira01.h

```diff
@@ -5,9 +5,20 @@
 
 struct BgTobira01;
 
+typedef void (*TobiraActionFunction)(struct BgTobira01*, PlayState*);
+
+#define DOOR_POS_MIN 0
+#define DOOR_POS_MAX 60
```

Comment:
Since those two constants are used only once I don't think they are important enough to have a macro